src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Components/RouteAttributeExtensionNode.cs

```diff
@@ -27,7 +27,7 @@ public override void WriteNode(CodeTarget target, CodeRenderingContext context)
         {
             context.CodeWriter.WriteLine("(");
             context.CodeWriter.WriteLine("// language=Route,Component");
-            using (context.CodeWriter.BuildEnhancedLinePragma(Source, context))
+            using (context.CodeWriter.BuildEnhancedLinePragma(Source, context, characterOffset: 1))
```

Comment:
What exactly would break if we passed 0 here today? Just trying to understand this better.

Reply:
@jjonescz Roslyn reports `error CS8938: The #line directive value is missing or out of range`